docs/runtime/core.md

```diff
@@ -1 +1,86 @@
 # Core
+
+Runtime Core is a compilation/execution engine for neural network models.
+
+## Modules
+
+Runtime Core has four modules. These are namespaces as well as directory names in `/runtime/onert/core/src/`.
+
+- `ir`  stands for Intermediate Representation which contains Neural Network Graph data structures
+- `compiler` converts IR to executable format
+- `exec` is execution module which is the result of compilation
+- `backend` is backend interface
+
+### Module `ir`
+
+This module contains data structures of pure Neural Network models. The models from NN Packages or NN API are converted to these structures.
+
+- `Subgraphs` is the entire neural network model which is a set of subgraphs
+- `Subgraph` consists of operands and operations
+- `Operand` (a.k.a. Tensor) has shape, data type, data and references to operations
+- `Operation` (a.k.a. Operator) has operation type, params and references to operands
+
+`Operand` and `Operation` are nodes of the graph, and the reference relationship between them is the edges of the graph.
+
+`Subgraphs` represents the whole model. It could have more than one `Subgraph` to support control flow operations. Those operations can make calls to another subgraph and when the execution on another subgraph is done it gets back to previous subgraph execution with returned operands.
```

Comment:
How about changing "to support control flow operations." to "branched by control flow operations."?

Reply:
I think we can leave it as the following sentence explains what "support" actually means...